person.rb

```diff
@@ -0,0 +1,17 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
- [ ] We only intend to retrieve `id` so let's therefore mark it with `attr_reader`, and then auto generate the `id` this way:

```ruby
class Person
    attr_accessor :name, :age 
  attr_reader :id 

  def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
 ...
end
```